### PR TITLE
Add email notification when auto offer is created

### LINKED ISSUE #913: Wishlist: email notifications - new template for notifications with autobid
related to: https://github.com/internetee/auction_center/issues/893

New template is described in automatic email document under the paragraph "Domeeninimi soovide nimekirjast jõudis oksjonile ja tehti automaatpakkumine"

please add new template to the system and use it when the automatic bid was made with wishlist entry

### PR DESCRIPTION
**Pull request by thiagoyoussef**
close #913

### CODE REVIEW

File: config/locales/mailers/wishlist_mailer.et.yml [hunk 1]
```diff
@@ -9,3 +9,9 @@ et:
         Oksjon lõppeb %{ends_at}.
       call_to_action_html: "Tee oma pakkumine %{link}."
       call_to_action_link: "siin"
+    auto_offer_notification_mail:
+      subject: Automaat pakkumine domeenile %{domain_name} oli edukas
+      salutation: "Lugupeetud %{display_name},"
+      introduction: Sooviloendis olev domeeninimi %{domain_name} pandi oksjonile ja tehti automaatne pakkumine %{offer_price}
+      call_to_action_html: "Vaadake oma pakkumist %{link}."
+      call_to_action_link: "siin"
```
Comment (thiagoyoussef) [inline config/locales/mailers/wishlist_mailer.et.yml:17]: Is the Estonian translation correct?

File: config/locales/mailers/wishlist_mailer.en.yml [hunk 2]
```diff
@@ -9,3 +9,9 @@ en:
         The auction ends at %{ends_at}.
       call_to_action_html: "Make your bid %{link}."
       call_to_action_link: "here"
+    auto_offer_notification_mail:
+      subject: Automatic bid for the %{domain_name} domain was successful"
+      salutation: "Dear %{display_name},"
+      introduction: The domain name %{domain_name} from the wishlist was auctioned and an automatic bid of %{offer_price} was made
+      call_to_action_html: "See your bid %{link}."
+      call_to_action_link: "here"
```
Comment (thiagoyoussef) [inline config/locales/mailers/wishlist_mailer.en.yml:17]: Any suggestions to improve the email description?

File: config/locales/mailers/wishlist_mailer.et.yml (hunk 1 above)
Comment (maricavor) [reply config/locales/mailers/wishlist_mailer.et.yml:17]: Automaatpakkumine is one word

**Comment by maricavor** [comment]
Can you cover with tests?

File: config/locales/mailers/wishlist_mailer.en.yml (hunk 2 above)
Comment (maricavor) [reply config/locales/mailers/wishlist_mailer.en.yml:17]: Instead of automatic use automated
If you use word 'here', make it a link

**Comment by thiagoyoussef** [comment]
I already created tests :)
Do you saw something specific that I forgot to test?

**Pull request merged**